src/components/Login.html

```diff
@@ -1,5 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
+	<head>
+		<link
+			rel="stylesheet"
+			href="../styles/root.css"
+		/>
+	</head>
+	<body>
+		<div
+			id="myModal"
+			class="inputDomMainContainer flex-column-center"
+		>
+			<form
+				class="formContainerMain inputDomContainer AlertInfo input-flex inputError box-shadow"
+				action="submit"
+				autocomplete="on"
+			>
+				<div
+					class="formContainer input-flex flex"
+				>
+					<div class="login-avatar">
+						<img
+							class="login-avatar-img round-border"
+							src="../images/steve.jpg"
+							alt=""
+						/>
+					</div>
+					<h3 class="h3Tag">Login</h3>
```

Comment:
you have given h3Tag to heading3. If its a universal for all the h3 tags try to give properties in CSS file directly to h3 tag and remove h3tag class